src/components/Personal.module.css

```diff
@@ -0,0 +1,26 @@
+.h1 {
+  /* text-align: center; */
+  font-style: italic;
+  font-size: 130%;
+  color: #00da91;
+  grid-column: 1/3;
+  grid-row: 1/1;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+.personalContainer {
+  display: grid;
```

Comment:
Good job using grid 👏
Also, you can take a look at MUI Grid component, can be useful: https://mui.com/material-ui/react-grid/